### DISCUSSION TITLE
Unfair bans in the TeamTalk 5 Official Server (US)

### DISCUSSION THREAD

**Discussion by jimmytruth** [BearWare/TeamTalk5#2041] [General]
Hello!

Yesterday around 6 PM EDT There was a guy by the username Mohammad joined the server banning anyone who has the nickname Flag of palestine or any nickname contains it.

It's so annoying that he's doing that. after talking to him it appeared that He's doing that for political reasons since he supports Israel and urging everyone who has Flag of palestine to either remove it or to replace it with flag of Israel, like simply he's controlling us.

It isn't fair of what's he's doing and that's against freedom of people to choose whatever nickname they like as long as it isn't  offending any person based on   race, ethnicity, caste, national origin, , sexual orientation, gender, gender identity, religious affiliation, age, disability, or serious disease or endity and seems he has no respect  for Democrasy, freedom of view and people's freedom to choose whatever they like.

Please (if possible) invalidate his account for good since what's he doing isn't right and the server is public for everyone to be there with different views and opinions.

**Comment by ziyadmohamed** [comment]
hello , i agree with that , and you should take actian as fast as you can

**Comment by ahmadsabbah12** [comment]
I strongly agree, he uses ban command for his personal opinion, and this is illegal in my opinion, I hope his account going to be removed from the admins list

**Comment by CoBC** [comment]
I just checked US server now and lot of people has "🇵🇸" in his 
nickname. SO not sure what's happened. Maybe @bear101 will have an idea 
on it

**Reply by jimmytruth** [reply]
He isn't available at the moment but I believe when he comes back he might ban all of those using it

**Reply by muamel1** [reply]
Simply these people are sympathetic to what's happening in Palestine in terms of oppression, but they were kicked out without any reason. I mean, everyone has their perspective and the angle from which they see things, but in the end, we should respect each other, shouldn't we?

**Comment by muamel1** [comment]
Yes, I was also kicked out just because my name has the Palestinian flag, even though I didn't harm anyone. I don't know when people will learn to respect different opinions and stop suppressing them.

**Comment by beqabeqa473** [comment]
just don't make provocations.
No one is interested in your or others mind. don't trigger others and
you won't be touched.
No need for expressing needless position moreover this aweful times.

On 10/10/23, Muamal Aljanahi ***@***.***> wrote:
> Simply these people are sympathetic to what's happening in Palestine in
> terms of oppression, but they were kicked out without any reason. I mean,
> everyone has their perspective and the angle from which they see things, but
> in the end, we should respect each other, shouldn't we?
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/BearWare/TeamTalk5/discussions/2041#discussioncomment-7234423
> You are receiving this because you are subscribed to this thread.
>
> Message ID:
> ***@***.***>


-- 
with best regards Beqa Gozalishvili
Tell: +995593454005
Email: ***@***.***
Web: https://gozaltech.org
Skype: beqabeqa473
Telegram: https://t.me/gozaltech
facebook: https://facebook.com/gozaltech
twitter: https://twitter.com/beqabeqa473
Instagram: https://instagram.com/beqa.gozalishvili

**Reply by muamel1** [reply]
Since when has using your Country flag in your name been provocative?

**Comment by CoBC** [comment]
I totally agree with Beqa. I think everyone, especially on a public 
server, have to be neutral about this type of topic.

For example the status message of an user called Dr. Ramon is: "fuck 
Israel", so this user has beeb banned due to this status message, not 
for palestine flag in the nickname.

**Reply by Ramon-Salazar** [reply]
@CoBC At that time because I got banned from the server because of the same reason, so I put the status, but the main reason was of the flag because that's what my main IP got banned for, though I removed it and already explained exactly the reasons of putting it, but again my another IP got banned by  the user Mohammad and he seems still banning people, and I think he banned user @Muamalaljanahi also for the same reason.

**Comment by jimmytruth** [comment]
As I know, he has been asked to remove the status and he did, but he got banned after that which isn't fair, so he needs to be unbanned immediately

**Comment by beqabeqa473** [comment]
when escalation is in all over the world, you should not use this to
trigger other people.
It is ok, if your country is neutral, but that's not your case.
And of course what you're saying is only empty words. all of us
understand why these things are doing and for that ban is required.
It is my personal opinion.
when i am seeing nicknames like free palestine and some
non-constructive smiles or words about israel, i will be forced to ban
that kind of people once and for a while. I would also automate
process for that purpose.
So my advice would be to stop such things and live in peace.


On 10/10/23, Muamal Aljanahi ***@***.***> wrote:
> Since when has using your Country flag in your name been provocative?
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/BearWare/TeamTalk5/discussions/2041#discussioncomment-7235424
> You are receiving this because you commented.
>
> Message ID:
> ***@***.***>


-- 
with best regards Beqa Gozalishvili
Tell: +995593454005
Email: ***@***.***
Web: https://gozaltech.org
Skype: beqabeqa473
Telegram: https://t.me/gozaltech
facebook: https://facebook.com/gozaltech
twitter: https://twitter.com/beqabeqa473
Instagram: https://instagram.com/beqa.gozalishvili

**Reply by muamel1** [reply]
You were not neutral in your comment, and you are asking people to be neutral. When you see an admin logging in with the name 'Fuck Palestine' and starting to provoke people on the server, don't expect anyone to be neutral. Please in the future try to make more sensible comments.

**Reply by Ramon-Salazar** [reply]
Exactly, and noticed that.

He logged in many times with the nickname Fuck Palestine, and I don't think we should be neutral about that since there are many sympathizers and I'm one of them, and really your opinion doesn't matter since you never came to the teamtalk5 official server (US).

**Comment by beqabeqa473** [comment]
I'm not going to follow this discussion with you. I stated an opinion.
I saw these nicknames with palestine flags on several servers and have
a reason telling that. If you don't want to understand, that's your
problem, but just in this case don't make noise.

On 10/10/23, Muamal Aljanahi ***@***.***> wrote:
> You were not neutral in your comment, and you are asking people to be
> neutral. When you see an admin logging in with the name 'Fuck Palestine' and
> starting to provoke people on the server, don't expect anyone to be neutral.
> Please in the future try to make more sensible comments.
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/BearWare/TeamTalk5/discussions/2041#discussioncomment-7240219
> You are receiving this because you commented.
>
> Message ID:
> ***@***.***>


-- 
with best regards Beqa Gozalishvili
Tell: +995593454005
Email: ***@***.***
Web: https://gozaltech.org
Skype: beqabeqa473
Telegram: https://t.me/gozaltech
facebook: https://facebook.com/gozaltech
twitter: https://twitter.com/beqabeqa473
Instagram: https://instagram.com/beqa.gozalishvili

**Reply by muamel1** [reply]
This is a lack of respect from you. Anyway, you don't know what happened, and you weren't there, so I don't think your opinion matters.
Have a nice day.
